packages/frontend/src/pages/Settings.tsx

```diff
@@ -143,16 +143,20 @@ const Settings: FC = () => {
                             exact
                             to={`${basePath}/appearance`}
                         />
-
-                        {localStorage.getItem('slack') && (
-                            <RouterMenuItem
-                                text="Slack"
-                                exact
-                                to={`${basePath}/slack`}
-                            />
-                        )}
                     </Menu>
                 </MenuWrapper>
+
+                {health.hasSlack && user.ability.can('manage', 'Organization') && (
+                    <MenuWrapper>
+                        <MenuHeader>Integrations</MenuHeader>
+
+                        <RouterMenuItem
```

Comment:
Seems to be missing a `<Menu>` component.

![Screenshot 2022-12-06 at 14 06 41](https://user-images.githubusercontent.com/9117144/205934135-a5c59059-2ff1-4096-b933-d5d425c9a892.png)